### ISSUE TITLE
Goreleaser Pro v2.4.8 was not added to releases-pro.json

### ISSUE THREAD

**Issue by mowies** [goreleaser/goreleaser#5314]
### What happened?

GoReleaser Pro v2.4.8 was released but not added to the [releases-pro.json](https://github.com/goreleaser/goreleaser/blob/main/www/docs/static/releases-pro.json) file yet.

### How can we reproduce this?

Renovate PR that wants to update to goreleaser-pro v2.4.8 but can't find the proper release: https://github.com/open-telemetry/opentelemetry-collector-releases/pull/752

### goreleaser version

```bash
v2.4.8
```

### GoReleaser Check

- [X] goreleaser check shows no errors

### Search

- [X] I did search for other open and closed issues before opening this

### Supporter

- [X] I am a [sponsor](https://github.com/sponsors/caarlos0/) or a [Pro](https://goreleaser.com/pro) customer

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by caarlos0** [comment]
hey, so sorry about this, not sure what happened.

while I investigate, manually ran the job: https://github.com/goreleaser/goreleaser/commit/b0a76aea5984aa64f1c2732e593eb864466c72f2

it should be there now!

**Comment by caarlos0** [comment]
ok, I think I know what happened.

Usually, I trigger both pro and OSS releases more or less together, and historically, goreleaser-pro finishes before OSS.

The pro release workflow was missing the generate trigger, so I never noticed this. When the OSS release workflow finished, it would update both.

This last time, the pro release took a little longer for some reason, and then we got this bug.

Fixed it there by adding the workflow-dispatch call.

Will keep an eye out for that in the future releases as well.

Thanks for the issue, and sorry for causing trouble, hopefully it wasn't too bad!

Cheers!
